### ISSUE TITLE
FaceAnalysis.__init__() got an unexpected keyword argument 'providers'

### ISSUE THREAD

**Issue by brundlesprout** [ZHO-ZHO-ZHO/ComfyUI-InstantID#19]
has anyone else seen this?

File "C:\SD\ComfyUI\ComfyUI\execution.py", line 155, in recursive_execute
    output_data, output_ui = get_output_data(obj, input_data_all)
                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\SD\ComfyUI\ComfyUI\execution.py", line 85, in get_output_data
    return_values = map_node_over_list(obj, input_data_all, obj.FUNCTION, allow_interrupt=True)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\SD\ComfyUI\ComfyUI\execution.py", line 78, in map_node_over_list
    results.append(getattr(obj, func)(**slice_dict(input_data_all, i)))
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\SD\ComfyUI\ComfyUI\custom_nodes\ComfyUI_IPAdapter_plus\IPAdapterPlus.py", line 543, in load_insight_face
    model = FaceAnalysis(name="buffalo_l", root=INSIGHTFACE_DIR, providers=[provider + 'ExecutionProvider',])
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Queue size: 0
Extra options

**Comment by whq1rgb** [comment]
I also encountered the same problem. Have you solved it？

**Comment by brundlesprout** [comment]
nope, trying reinstall. still no luck. maybe clean comfyi install? stumped

**Comment by wenshaoru** [comment]
same problem...

**Comment by dylan1951** [comment]
bad insightface version

**Comment by slayersbosh** [comment]
> same problem...

How to solve this?
